README.md

```diff
@@ -13,10 +13,27 @@ See [Stockpile API](https://github.com/AdamVig/stockpile-api) for the Restful AP
 - Clone this repo
 - Make sure you have the latest version of Ionic and Cordova installed `npm install -g ionic cordova`
 - Run `npm install` from the project root
+
+**Note:** If your build is slow, update `npm` to 3.x: `npm install -g npm`.
+
+## Running in a Browser
+Running in the browser is very simple. However, running the app in `localhost` causes CORS issues. You can install an extension in your browser and allow them for `https://stockpile.adamvig.com` while we fix the issue with the proxy.
+
 - Run `ionic serve` to preview the app in your browser
 - You can also run `ionic serve --lab` to preview different platforms side-by-side
 
-**Note:** If your build is slow, update `npm` to 3.x: `npm install -g npm`.
+**Note:** Some native functionality like scanning won't be available in the browser. You have to run the app on an emulator or device for the full experience.
+
+## Running on a Device or Emulator
+Running apps on a device or emulator requires you to build the app on your computer. To do this, you have to set up the Android SDK and Apple's Xcode. For detailed instructions on how to do this, follow [this guide for Mac](https://ionicframework.com/docs/v2/resources/platform-setup/mac-setup.html) or [this one for Windows](https://ionicframework.com/docs/v2/resources/platform-setup/windows-setup.html).
+
+- Run `ionic platform add ios` and/or `ionic platform add android`
+- Run `ionic plugin add phonegap-plugin-barcodescanner`
```

Comment:
I thought it would be too, but the only thing that changes when I add a plugin is the `/plugin` directory that's not committed to GitHub, so it looks like you have to add the plugins manually when you clone the repo.